### PR TITLE
Adding myself to contributor's page

### PR DESCRIPTION
**Pull request by Shivangibansal**

### CODE REVIEW

**Comment by Macbull** [comment]
1. Please close this pull request temporarily for now
2. do the required changes to contributors/home.html and crowdsource.routes.js
3. commit those changes
4. push changes to your fork
5. reopen this pull request (it will be updated automatically after push)

**Comment by dmorina** [comment]
resolve conflicts and remove staticfiles/js/crowdsource.routes.js~

File: staticfiles/js/crowdsource.routes.js~ [hunk 1]
```diff
@@ -0,0 +1,166 @@
+(function () {
```
Comment (nistala) [inline staticfiles/js/crowdsource.routes.js~]: please remove temp file

**Comment by Shivangibansal** [comment]
How can I remove the temp file(staticfiles/js/crowdsource.routes.js~)? I cannot find it in my local repo anymore.

**Comment by nistala** [comment]
```git rm staticfiles/js/crowdsource.routes.js~```
regardless of if it's in the repo or not

**Comment by neilthemathguy** [comment]
Taking this PR out of the pending list.  Please fix the bugs/resolve conflicts and raise the new PR or move to the Milestone 2. Thank you.

**Pull request closed without merging**